Fasting App/Extensions/Extensions.swift

```diff
@@ -353,3 +353,54 @@ extension ActivityPickerView: UIPickerViewDataSource, UIPickerViewDelegate {
         
     }
 }
+
+extension UIViewController {
+    func setBackground() {
+        
+        let layer = CAGradientLayer()
+        guard
+            let mainColor = UIColor.stdBackground,
+            let topColor = UIColor.topBackground else { return }
+        
+        layer.frame = view.bounds
+        layer.colors = [mainColor.cgColor, topColor.cgColor]
+        layer.startPoint = CGPoint(x: 0, y: 0)
+        layer.endPoint = CGPoint(x: 1, y: 1)
+        view.layer.insertSublayer(layer, at: 0)
+    }
+}
```

Comment:
Create extension files for each type of extension, for example

`UIViewController+Extensions.swift`

and this will contain all you're UIViewController extensions.

It is also good to comment on extensions functions to explain what they do.

Reply:
Done